src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fixed the issue that custom URIs are not supported for context creation [#17285]
```

Comment:
Might should be : "Supported create storage context with customized blob, queue, file, table service endpoint."

Reply:
Done